server/src/main/java/org/elasticsearch/common/lucene/index/ElasticsearchLeafReader.java

```diff
@@ -50,7 +52,7 @@ public ShardId shardId() {
     }
 
     @Override
-    public CacheHelper getCoreCacheHelper() {
+    public IndexReader.CacheHelper getCoreCacheHelper() {
```

Comment:
I wonder why this change is needed as the related ES commit does not change anything here?